lib/tlscontext.c

```diff
@@ -66,6 +66,10 @@ struct _TLSContext
   gboolean ocsp_stapling_verify;
 
   SSL_CTX *ssl_ctx;
+#if SYSLOG_NG_HAVE_DECL_SSL_CONF_CMD
+  SSL_CONF_CTX *ssl_conf_ctx;
+  GList *conf_cmds_list;
+#endif
```

Comment:
let's avoid making this conditional.

Reply:
I'll remove all the conditionals, we had a round around this with MrAnno, just wanted to see your opinions too.

Reply:
cool